dashboard/package.json

```diff
@@ -110,20 +111,19 @@
     "http-proxy-middleware": "^0.18.0",
     "intl": "^1.2.5",
     "intl-relativeformat": "^2.1.0",
-    "jest": "^22.4.3",
-    "jest-enzyme": "^6.0.0",
-    "jest-fetch-mock": "^1.5.0",
-    "jest-localstorage-mock": "^2.2.0",
+    "jest": "^23.6.0",
```

Comment:
Updating to `^25.4` would have required updating `babel` as well, which, I felt was out of the scope of this PR.

Reply:
Upgrading to babel 7 is something I would like to do ASAP tho. Doing so shouldn't cause us any problems.